users/reverseproxy.rst

```diff
@@ -46,6 +46,21 @@ Nginx
       proxy_pass              http://localhost:8384/;
     }
 
+Caddy
+~~~~~
+
+.. code-block:: nginx
```

Comment:
Right, with the GitHub preview it looked ok but I see in the compiled docs it doesn't work.